### PR TITLE
Enable node cleaning by default for OSP > 13

### PR DESCRIPTION
**Pull request by smalleni**
Signed-off-by: Sai Sindhur Malleni <smalleni@redhat.com>

### CODE REVIEW

**Comment by smalleni** [comment]
@asyedham @venkataanil PTAL

**Review by akrzos — APPROVED** [review]
LGTM

File: templates/undercloud.conf.j2 [hunk 1]
```diff
@@ -11,6 +11,7 @@ container_insecure_registries={{ registry_mirror }},{{ insecure_registries }}
 undercloud_timezone = UTC
 undercloud_public_host = {{ undercloud_public_host }}
 undercloud_admin_host = {{ undercloud_admin_host }}
+clean_nodes = true
```
Comment (asyedham) [inline templates/undercloud.conf.j2]: Instead, we can set the clean_debug to true by default in [group_vars/all.yml](https://github.com/redhat-performance/jetpack/blob/master/group_vars/all.yml#L121) and disable the node cleaning if needed by setting it to false.
This way there is no need for setting clean_nodes to true again. Thoughts?

**Review by asyedham — CHANGES_REQUESTED** [review]

File: templates/undercloud.conf.j2 (hunk 1 above)
Comment (smalleni) [reply templates/undercloud.conf.j2]: Oh sweet, I didn’t even check that we already have that variable. Yeh let me just change the default there. Good point.

File: group_vars/all.yml [hunk 2]
```diff
@@ -118,7 +118,7 @@ inspection_iprange: 192.168.24.110,192.168.24.250
 # undercloud to access overcloud resources
 external_gateway: 172.17.5.1/24
 external_network_vlan_id: 300
-clean_debug: False
+clean_nodes: True
```
Comment (masco) [inline group_vars/all.yml:121]: can you change this on CI vars also so you will get a green mark from CI ;-)

Comment (smalleni) [reply group_vars/all.yml:121]: Sure, wasn't sure if you wanted node cleaning in CI :)

**Comment by smalleni** [comment]
Done with requested changes.

**Review by masco — APPROVED** [review]
waiting for CI result

**Pull request merged**